tests/integration/modules/test_vsphere.py

```diff
@@ -52,3 +53,18 @@ def test_list_ssds(service_instance):
     for host_name in ret:
         for ssd in ret[host_name]:
             assert isinstance(ssd, str)
+
+
+def test_list_non_ssds(service_instance):
+    pattern = r"^[a-zA-Z0-9]{3}\."
+    ret = vsphere.list_non_ssds(
+        service_instance=service_instance,
+    )
+    assert ret
+    for host_name in ret:
+        for ssd in ret[host_name]:
```

Comment:
I think nonssd might be a better variable name here, gives quicker flow into the mind, otherwise have to do double-take with ssd , wait I wanted non-ssd, oh! wait etc.